src/detect-bytejump.c

```diff
@@ -581,6 +581,7 @@ static int DetectBytejumpSetup(DetectEngineCtx *de_ctx, Signature *s, const char
         if (bed_sm == NULL) {
             SCLogError(SC_ERR_INVALID_SIGNATURE, "Unknown byte_extract var "
                        "seen in byte_jump - %s\n", offset);
+            SCFree(offset);
```

Comment:
Should this be freed in preceding `goto error` cases?

Reply:
Thank you @jlucovsky 
This is the only `goto` within the condition where `if (offset != NULL) {` at line 579
And it gets freed at line 589 if there is no error 
So I think we only need this one.
Does it look good to you so ?